app/common/lib/suggestion.js

```diff
@@ -196,30 +195,30 @@ const createVirtualHistoryItems = (historySites, urlLocationLower) => {
   // parse each history item
   const parsedHistorySites = []
   historySites.forEach((site) => {
-    if (site && site.location) {
+    if (site.get && site.get('location')) {
       parsedHistorySites.push(
-        urlParse(site.location)
+        urlParse(site.get('location'))
```

Comment:
nitpick: this might be better like (to avoid calling get twice):
```
const siteLocation = site.get && site.get('location')
if (siteLocation) {
  parsedHistorySites.push(
    urlParse(siteLocation)
  )
}